jime.csl

```diff
@@ -0,0 +1,362 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of Interactive Media in Education - Harvard</title>
+    <title-short>JIME - Harvard</title-short>
+    <id>http://www.zotero.org/styles/jime</id>
+    <link href="http://www.zotero.org/styles/jime" rel="self"/>
```

Comment:
Please change the ID and and self link in lines 6+7 to the following according to our [submission guidelines](**url**):
..../harvard-journal-of-interactive-media-in-education   (instead of the .../jime)

Please also change the actual file name to match that, with the .csl suffix

Reply:
Addressed via 3ec264f0095d0a4fa746917d82b343738a23513f